internal/test/e2e/run.go

```diff
@@ -83,27 +83,27 @@ func RunTestsInParallel(conf ParallelRunConf) error {
 }
 
 type instanceRunConf struct {
-	amiId, instanceProfileName, storageBucket, jobId, parentJobId, subnetId, regex, instanceId string
-	bundlesOverride                                                                            bool
+	amiId, instanceProfileName, storageBucket, jobId, parentJobId, commandId, subnetId, regex, instanceId string
```

Comment:
I'm not sure `commandId` makes a lot of sense in this struct
This is supposed to be the input for an instance run. The command id seems more like the output

I think it would make more sense in `instanceTestsResults`